### PR TITLE
File download issue fix

### PR DESCRIPTION
**Pull request by rleojoseph**
# Description
Fix for downloading stringified base64 data and launching Activity controller

## Links
MINI-4332

# Checklist
- [x] I have read the [contributing guidelines](CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `fastlane ci` without errors

### CODE REVIEW

File: Example/Controllers/Extensions/ViewController+MiniApp.swift [hunk 1]
```diff
@@ -15,8 +15,11 @@ extension ViewController: MiniAppNavigationDelegate {
                            with responseHandler: @escaping MiniAppNavigationResponseHandler,
                            onClose closeHandler: MiniAppNavigationResponseHandler?) {
         if url.absoluteString.starts(with: "data:") {
+            var base64String = ""
             // currently js sdk is passing no base64 data type
-            let base64String = url.absoluteString.replacingOccurrences(of: "data:;base64,", with: "")
+            if url.absoluteString.contains("data:image/jpeg;base64") {
+                base64String = url.absoluteString.replacingOccurrences(of: "data:image/jpeg;base64", with: "")
```
Comment (corycaywood) [inline Example/Controllers/Extensions/ViewController+MiniApp.swift:21]: Will this work for other file types other than images? Also, I think the `,` needs to be removed too?

Actually in the case of Android, we just split the string on the `,` and then only decode the portion afterwards. Would that work here?

Comment (rleojoseph) [reply Example/Controllers/Extensions/ViewController+MiniApp.swift]: Done

**Review by corycaywood — APPROVED** [review]
Looks good 👍

**Review by Climbatize — APPROVED** [review]
good 👍

**Pull request merged**